packages/catalog-model/examples/acme/org.yaml

```diff
@@ -3,6 +3,9 @@ kind: Group
 metadata:
   name: acme-corp
   description: The acme-corp organization
+  links:
```

Comment:
Maybe we should have at least one example with more than one link? Except I missed it 😁